### PR TITLE
First part of the checkstyle fix. 

### PR DESCRIPTION
**Pull request by sima-zhu**
The PR is the checkstyle fix for storage. #3021 
All autorest file bugs documented in #3506 
Suppressed the Autorest files. Make changes to part of the files.
Major issues in files are: import, indentation, operator should be in new line.

There will be another PR for the rest of the checkstyle fix.

### CODE REVIEW

File: storage/data-plane/src/main/java/com/microsoft/azure/storage/blob/GeneratedAppendBlobs.java [hunk 1]
```diff
@@ -32,10 +32,10 @@
 import io.reactivex.Flowable;
 import io.reactivex.Single;
 import io.reactivex.annotations.NonNull;
+
```
Comment (g2vinay) [inline storage/data-plane/src/main/java/com/microsoft/azure/storage/blob/GeneratedAppendBlobs.java]: don't need the newline.

File: storage/data-plane/src/main/java/com/microsoft/azure/storage/blob/GeneratedBlobs.java [hunk 2]
```diff
@@ -50,10 +50,10 @@
 import io.reactivex.Maybe;
 import io.reactivex.Single;
 import io.reactivex.annotations.NonNull;
+
```
Comment (g2vinay) [inline storage/data-plane/src/main/java/com/microsoft/azure/storage/blob/GeneratedBlobs.java]: don't need the newline.

File: storage/data-plane/src/main/java/com/microsoft/azure/storage/blob/GeneratedBlockBlobs.java [hunk 3]
```diff
@@ -38,10 +38,10 @@
 import io.reactivex.Maybe;
 import io.reactivex.Single;
 import io.reactivex.annotations.NonNull;
+
```
Comment (g2vinay) [inline storage/data-plane/src/main/java/com/microsoft/azure/storage/blob/GeneratedBlockBlobs.java]: don't need the newline.

File: storage/data-plane/src/main/java/com/microsoft/azure/storage/blob/GeneratedPageBlobs.java [hunk 4]
```diff
@@ -42,10 +42,10 @@
 import io.reactivex.Maybe;
 import io.reactivex.Single;
 import io.reactivex.annotations.NonNull;
+
```
Comment (g2vinay) [inline storage/data-plane/src/main/java/com/microsoft/azure/storage/blob/GeneratedPageBlobs.java]: don't need the newline.

File: storage/data-plane/src/main/java/com/microsoft/azure/storage/blob/GeneratedAppendBlobs.java [hunk 5]
```diff
@@ -35,7 +35,6 @@
 import java.net.URL;
 import java.nio.ByteBuffer;
 import java.time.OffsetDateTime;
-import java.util.HashMap;
```
Comment (rickle-msft) [inline storage/data-plane/src/main/java/com/microsoft/azure/storage/blob/GeneratedAppendBlobs.java]: This file and the other files prefixed with Generated are also output from Autorest, so they should also be excluded from the checkstyle report.

Comment (sima-zhu) [reply storage/data-plane/src/main/java/com/microsoft/azure/storage/blob/GeneratedAppendBlobs.java]: Good catch

File: storage/data-plane/src/test/java/com/microsoft/azure/storage/Samples.java [hunk 6]
```diff
@@ -471,11 +528,11 @@ public void exampleBlobURLParts() throws MalformedURLException, UnknownHostExcep
...
          (SAS).
          */
-        URL u = new URL("https://myaccount.blob.core.windows.net/mycontainter/ReadMe.txt?" +
-                "snapshot=2011-03-09T01:42:34.9360000Z" +
-                "&sv=2015-02-21&sr=b&st=2111-01-09T01:42:34Z&se=2222-03-09T01:42:34Z&sp=rw" +
-                "&sip=168.1.5.60-168.1.5.70&spr=https,http&si=myIdentifier&ss=bf&srt=s" +
-                "&sig=92836758923659283652983562==");
+        URL u = new URL("https://myaccount.blob.core.windows.net/mycontainter/ReadMe.txt?"
+                + "snapshot=2011-03-09T01:42:34.9360000Z"
+                + "&sv=2015-02-21&sr=b&st=2111-01-09T01:42:34Z&se=2222-03-09T01:42:34Z&sp=rw"
```
Comment (g2vinay) [inline storage/data-plane/src/test/java/com/microsoft/azure/storage/Samples.java]: Using StringBuilder would be near and efficient here

File: storage/data-plane/src/test/java/com/microsoft/azure/storage/Samples.java [hunk 7]
```diff
@@ -1708,11 +1765,11 @@ Create the blob with string (plain text) content.
...
```
Comment (g2vinay) [inline storage/data-plane/src/test/java/com/microsoft/azure/storage/Samples.java]: Same, StringBuilder might be a neat and efficient choice for constructing this string

File: storage/data-plane/src/test/java/com/microsoft/azure/storage/Samples.java (hunk 6 above)
Comment (sima-zhu) [reply storage/data-plane/src/test/java/com/microsoft/azure/storage/Samples.java]: I see your points, thanks for pointing this out. 
I prefer not to change right here for 2 reasons:
1. This is checkstyle change, so the improvement is kind of unrelated to the PR itself.
2. For StringBuilder, the Java JVM has the ability to optimize the code automatically, so it is not big concern of efficiency. Refer to this [one](https://stackoverflow.com/questions/4645020/when-to-use-stringbuilder-in-java).
My personal preference is not to introduce more structures to make code clean.

File: storage/data-plane/src/test/java/com/microsoft/azure/storage/Samples.java (hunk 7 above)
Comment (sima-zhu) [reply storage/data-plane/src/test/java/com/microsoft/azure/storage/Samples.java]: Same reason above.

**Pull request merged**